addons/analytic/__openerp__.py

```diff
@@ -18,10 +18,10 @@
     'data': [
         'security/analytic_security.xml',
         'security/ir.model.access.csv',
-        'views/analytic_view.xml',
+        'views/analytic_account_views.xml',
```

Comment:
Actually I would name it account_analytic_views.xml (models are account_analytic_* ).